src/scss/abstracts/mixins/_forms.scss

```diff
@@ -0,0 +1,22 @@
+// sizes passed to the input-width function; they should roughly correspond to the number of characters (e.g. input-width 4 = 4 characters).
+// Specific values were taken from the DST design system
+$input-widths: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 50;
+
+// default character width value for mixin
+$char-max-width: 0.96rem;
```

Comment:
I was hoping to use ch instead of rem, but the unit's implementation across browsers (specifically IE11 vs the rest) meant the calculations were done differently, and a tolerance would likely need to be added when working on creating input widths of a 'set' character length